tests/test_003-config.py

```diff
@@ -186,8 +186,15 @@ def test_load_config():
     t.eq(app.cfg.workers, 3)
     t.eq(app.cfg.proc_name, "fooey")
 
+def test_load_config_explicit_file():
+    with AltArgs(["prog_name", "-c", "file:%s" % cfg_file()]):
+        app = NoConfigApp()
+    t.eq(app.cfg.bind, ["unix:/tmp/bar/baz"])
+    t.eq(app.cfg.workers, 3)
+    t.eq(app.cfg.proc_name, "fooey")
+
 def test_load_config_module():
-    with AltArgs(["prog_name", "-c", cfg_module()]):
+    with AltArgs(["prog_name", "-c", "python:%s" % cfg_module()]):
```

Comment:
I think it would be nice to keep the test untouched to make sure that we don't break backwards compatibility. We can add a separate test for the `python:` prefix.